README.md

```diff
@@ -84,33 +84,28 @@ Relayer:
 |--------------------------------------------------|-------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|
 | `RELAYER_NEUTRON_CHAIN_RPC_ADDR`                 | `string`          | rpc address of neutron chain                                                                                                                                               | required |
 | `RELAYER_NEUTRON_CHAIN_REST_ADDR`                | `string`          | rest address of neutron chain                                                                                                                                              | required |
-| `RELAYER_NEUTRON_CHAIN_CHAIN_ID `                | `string`          | neutron chain id                                                                                                                                                           | required |
 | `RELAYER_NEUTRON_CHAIN_HOME_DIR   `              | `string`          | path to keys directory                                                                                                                                                     | required |
 | `RELAYER_NEUTRON_CHAIN_SIGN_KEY_NAME`            | `string`          | key name                                                                                                                                                                   | required |
 | `RELAYER_NEUTRON_CHAIN_TIMEOUT `                 | `time`            | timeout of neutron chain provider                                                                                                                                          | required |
```

Comment:
This config value is not required anymore (and some other variables as well), please update README accordingly.

Reply:
I've changed these fields with default to `optional`

Reply:
Thank you!